R/pkg/R/stats.R

```diff
@@ -135,13 +136,13 @@ setMethod("freqItems", signature(x = "SparkDataFrame", cols = "character"),
 #' Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
 #'
 #' The result of this algorithm has the following deterministic bound:
-#' If the SparkDataFrame has N elements and if we request the quantile at probability `p` up to
-#' error `err`, then the algorithm will return a sample `x` from the SparkDataFrame so that the
-#' *exact* rank of `x` is close to (p * N). More precisely,
-#'   floor((p - err) * N) <= rank(x) <= ceil((p + err) * N).
-#' This method implements a variation of the Greenwald-Khanna algorithm (with some speed
-#' optimizations). The algorithm was first present in [[http://dx.doi.org/10.1145/375663.375670
-#' Space-efficient Online Computation of Quantile Summaries]] by Greenwald and Khanna.
+#' If the SparkDataFrame has N elements and if we request the quantile at probability \strong{p} up
```

Comment:
@shivaram  Looking at the doc page for statfunctions, a lot of functions are being mushed together.  E.g., "col1" and "col2" appear under the "Arguments" section many times.  What is the best way to separate these methods?